tchannel/schemes/__init__.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+RAW = 'raw'
+JSON = 'json'
+THRIFT = 'thrift'
+DEFAULT = RAW
+
+DEFAULT_NAMES = (
+    RAW,
+    JSON,
+    THRIFT
+)
+
+from .raw import RawArgScheme  # noqa
+from .json import JsonArgScheme  # noqa
+from .thrift import ThriftArgScheme  # noqa
```

Comment:
yeah - you want to tame import order in a given packages init. In this case, the *Scheme types use the constants defined above them.